sigstore-java/src/main/java/dev/sigstore/encryption/Keys.java

```diff
@@ -74,7 +74,7 @@ public static PublicKey parsePublicKey(byte[] keyBytes)
       throw new InvalidKeySpecException("Invalid key, could not parse PEM section");
     }
     // special handling for PKCS1 (rsa) public key
-    if (section == null) {
+    if ((section == null) || (section.getContent() == null)) {
       throw new InvalidKeySpecException("Invalid key");
```

Comment:
while we're in here, can we just add in "Invalid key, empty section" or something to the message?

Reply:
Changed accordingly.